src/panel/pages/timeline/components/TimelineEvent.fixture.tsx

```diff
@@ -64,5 +65,12 @@ export default {
     <Wrapper>
       <TimelineEvent event={teardownEvent} />
     </Wrapper>
+  ),
+  "event with tooltip": (
```

Comment:
Because we have individual tooltips fixtured below, I don't think we need to force state in this way.

There's two ways we could view a tootip in fixtures with what you've got:
 - View the tooltip component
 - Hover on a component which has tooltip-on-hover functionality